sbibm/tasks/__init__.py

```diff
@@ -63,6 +63,11 @@ def get_task(task_name: str, *args: Any, **kwargs: Any) -> Task:
 
         return TwoMoons(*args, **kwargs)
 
+    elif task_name == "norefposterior":
```

Comment:
Can we come up with a more descriptive name for the task?

Reply:
Not sure, I can tell you what it is supposed to mimick: a xray beam hitting a beam monitor screen. Did that help? :-D 

I basically hoped to keep it general, as I believe the contents are not that important. There are and will be so many problems out there which offer unreachable reference posteriors, that my idea was more to use this task for demonstration purposes.

Reply:
Yes, that actually helped understanding it :) I agree that knowing the exact problem is not necessary for the benchmark task but it's still nice to know what its inspired by/might relate to. Could you add your sentence to the class docstring on the `__init__` method?

I still think `norefposterior` is a bit too generic, since we might want to add other tasks without reference posterior in the future. How about `xray_noref`, `beam_noref` or anything along those lines?

Reply:
I like `noref_beam`, see 9fda640